doc/reference/storage_lvm.md

```diff
@@ -1,45 +1,62 @@
 (storage-lvm)=
 # LVM - `lvm`
 
- - Uses LVs for images, then LV snapshots for instances and instance snapshots.
- - The filesystem used for the LVs is ext4 (can be configured to use xfs instead).
- - By default, all LVM storage pools use an LVM thinpool in which logical
-   volumes for all LXD storage entities (images, instances, etc.) are created.
-   This behavior can be changed by setting "lvm.use\_thinpool" to "false". In
-   this case, LXD will use normal logical volumes for all non-instance
-   snapshot storage entities (images, instances, etc.). This means most storage
-   operations will need to fallback to rsyncing since non-thinpool logical
-   volumes do not support snapshots of snapshots. Note that this entails
-   serious performance impacts for the LVM driver causing it to be close to the
-   fallback DIR driver both in speed and storage usage. This option should only
-   be chosen if the use-case renders it necessary.
- - For environments with high instance turn over (e.g continuous integration)
-   it may be important to tweak the archival `retain_min` and `retain_days`
-   settings in `/etc/lvm/lvm.conf` to avoid slowdowns when interacting with
-   LXD.
-
-## Storage pool configuration
+{abbr}`LVM (Logical Volume Manager)` is a storage management framework rather than a file system.
+It is used to manage physical storage devices, allowing you to create a number of logical storage volumes that use and virtualize the underlying physical storage devices.
+
+Note that it is possible to over-commit the physical storage in the process, to allow flexibility for scenarios where not all available storage is in use at the same time.
+
+## Terminology
+
+LVM can combine several physical storage devices into a *volume group*.
+You can then allocate *logical volumes* of different types from this volume group.
+
+One supported volume type is a *thin pool*, which allows over-committing the resources by creating volumes that are larger than the available physical storage.
+Another type is a *volume snapshot*, which captures a specific state of a logical volume.
+
+## `lvm` driver in LXD
+
+The `lvm` driver in LXD uses logical volumes for images, and volume snapshots for instances and snapshots.
+The file system used for the logical volumes is ext4 by default.
+You can configure the `lvm` driver to use XFS instead by setting the {ref}`block.filesystem <storage-lvm-vol-config>` configuration for the volume (or the corresponding `volume.block.filesystem` configuration on the storage pool for all volumes in the pool).
+
+By default, LVM storage pools use an LVM thin pool and create logical volumes for all LXD storage entities (images, instances etc.) in there.
+This behavior can be changed by setting {ref}`lvm.use_thinpool <storage-lvm-pool-config>` to `false`.
```

Comment:
"This behaviour can be changed at pool create time..."

It cannot be changed later.